src/Initializer/InputParameters.hpp

```diff
@@ -104,6 +105,11 @@ struct MeshParameters {
   bool showEdgeCutStatistics;
 };
 
+struct CubeGeneratorParameters {
```

Comment:
Why enclose this into a struct on its own? Maybe `seissol::geometry::CubeGeneratorParameters` works already here?

Reply:
I enclosed it into a seperate struct to preserve the design symmetry of the struct `SeisSolParameters`. If that doesn't matter, I can adjust `SeisSolParameters` to use `seissol::geometry::CubeGeneratorParameters` directly.